server/server.js

```diff
@@ -75,6 +77,14 @@ app.get(`${apiBaseUrl}/employer-form`,
     return res.json({ data: result });
   }));
 
+app.get(`${apiBaseUrl}/employer-form/:id`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
```

Comment:
IMO we should still check the region here in case of HA - otherwise it could be possible for a user to navigate to a URL directly and see something they shouldn't have access to

Reply:
I'm unsure about how to resolve this. Would it require rewriting of the getEmployerByID function?

Reply:
I can think of two options offhand:

1. Get the site by ID, then compare here the site region vs. the user's regions and decide what to return accordingly
2. Add the user's regions as a param to this method, and then include it as part of the WHERE clause in the query - so a site only comes back from the DB if the HA is in the list provided *and* the ID matches

I feel like #2 is bulletproof but I'm not sure if it confuses things by moving some business logic into the DB access layer instead of the API endpoints

Reply:
Another option would be to 
1. Add user regions to `req.userInto` object based on sites if HA, etc.
2. Check if `req.userInfo.regions` contains the region we're fetching (would fit well where we tend to do validation)